interfaces/builtin/avahi_control.go

```diff
@@ -124,25 +124,29 @@ func (iface *avahiControlInterface) AppArmorConnectedPlug(spec *apparmor.Specifi
 	} else {
 		new = slotAppLabelExpr(slot)
 	}
-	snippet := strings.Replace(avahiObserveConnectedPlugAppArmor+avahiControlConnectedPlugAppArmor, old, new, -1)
+	snippet := strings.Replace(avahiControlConnectedPlugAppArmor+avahiControlConnectedPlugAppArmor, old, new, -1)
```

Comment:
Well, this is *avahi_control* and otherwise the *control* snippet was just unused. Perhaps the rules need tweaking but this was a clear mistake IMO.